components/GlobalTags.tsx

```diff
@@ -11,6 +11,10 @@ function GlobalTags() {
 
       {/* Web Manifest */}
       <link rel="manifest" href={asset("/site.webmanifest")} />
+
+      <script>
```

Comment:
I'd better move it to a GTM  app on decohub

Reply:
It was added temporarily to have access to the dataLayer and perform test